src/Maestro/SubscriptionActorService/MaestroBarClient.cs

```diff
@@ -237,5 +237,28 @@ public Task<Channel> GetChannelAsync(int channelId)
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
```

Comment:
These comments are not true. This method doesn't do any of what these comments say it does. They, in fact, just throw.